tests/backends/hardware/sr/v4/test_power_board.py

```diff
@@ -0,0 +1,470 @@
+"""Test the SR v4 PowerBoard backend and associated classes."""
+
+import struct
+from datetime import timedelta
+from typing import List, Optional, Type, Union
+
+import pytest
+import usb
+
+from j5.backends import Backend
+from j5.backends.hardware.sr.v4.power_board import (
+    CMD_READ_5VRAIL,
+    CMD_READ_BATTERY,
+    CMD_READ_BUTTON,
+    CMD_READ_FWVER,
+    CMD_READ_OUTPUT,
+    CMD_WRITE_ERRORLED,
+    CMD_WRITE_OUTPUT,
+    CMD_WRITE_PIEZO,
+    CMD_WRITE_RUNLED,
+    ReadCommand,
+    SRV4PowerBoardHardwareBackend,
+    USBCommunicationError,
+    WriteCommand,
+    handle_usb_error,
+)
+from j5.boards import Board
+from j5.boards.sr.v4.power_board import PowerBoard, PowerOutputPosition
+from j5.components.piezo import Note
+
+
+def test_read_command():
+    """Test that ReadCommand behaves as expected."""
+    rc = ReadCommand(1, 2)
+    assert issubclass(ReadCommand, tuple)
```

Comment:
I don't forsee us using the `tuple` properties of `ReadCommand` or `WriteCommand`, so I doubt this test is needed. I don't feel strongly about this though.